### DISCUSSION TITLE
Incorrect answer to Question 1 in Course 3 (page 27/34)

### DISCUSSION THREAD

**Discussion by ave2a** [TabularEditor/TabularEditor3#1235] [Learn]
To get the answer to **Question 1** in **Course 3** correct, I have to choose True  for **There are 20 tables in the model and over 200 columns**. I think this is incorrect, because Tabular Editor shows a number of 186 columns. Is this anwer incorrect or do I think wrong?

**Comment by otykier** [comment]
Thanks for bringing this to our attention, and our apologies for any inconvenience caused. We have corrected the question now.
